docs/deploy/howto-register-aad-applications.md

```diff
@@ -1,21 +1,141 @@
-# AAD App Registration
+# AAD App Registration  <!-- omit in toc -->
 
 [Home](readme.md)
 
-Components of Azure Industrial IoT solution require several App Registrations in your Azure Active Directory (AAD). Based on type of your deployment those will be either two or three App Registrations.
+## Table Of Contents <!-- omit in toc -->
 
-* ServicesApp
-* ClientsApp
-* App registration for AKS cluster.
+* [Introduction](#introduction)
+* [PowerShell scripts](#powershell-scripts)
+* [Microsoft.Azure.IIoT.Deployment](#microsoftazureiiotdeployment)
 
-## ServicesApp
+## Introduction
 
-TBD
+Components of Azure Industrial IoT solution require several App Registrations in your Azure Active Directory
+(AAD) to run. Those App Registrations are responsible for:
 
-## ClientsApp
+* providing identity for back-end microservices to run
+* defining authentication methods and details for client applications
 
-TBD
+Both of our deployment methods (`deploy.ps1` script or `Microsoft.Azure.IIoT.Deployment` application) create
+those App Registrations for you by default. But they both require user to have `Administrator` role in the AAD
```

Comment:
missing "the"

Reply:
the `deploy.ps1` script or the `Microsoft.Azure.IIoT.Deployment` application, right ?

Reply:
Fixed, added the in front of `Microsoft.Azure.IIoT.Deployment` application.